### PR TITLE
Fix mcp concern syntax hint

### PR DESCRIPTION
**Pull request by apiraino**
Fixing a typo when suggesting the syntax to raise concerns on compiler MCPs

### CODE REVIEW

File: src/compiler/mcp.md [hunk 1]
```diff
@@ -137,14 +137,14 @@ These types of procedural comments can be left on the issue (it's also good to l
 previous section. To facilitate a machine parsable scanning of the concerns please use the following syntax to formally
 register a concern:
 ```
-@rustbot concern reason-for-concern
+@rfcbot concern reason-for-concern
```
Comment (jieyouxu) [inline src/compiler/mcp.md:140]: Question: isn't this only applicable for *FCP*s, does rfcbot (not to be confused with rustbot) handle MCPs now?

Comment (apiraino) [reply src/compiler/mcp.md:140]: always has been the case, [AFAICT](https://github.com/rust-lang/triagebot/blob/4b28086e6b7e7566534a8e936ad9b9acf6105a6c/src/github.rs#L2117)

Comment (jieyouxu) [reply src/compiler/mcp.md:140]: Oh cool, did not know that also applied to non-FCP/RFC, i.e. MCP

**Review by jieyouxu — APPROVED** [review]
Thanks!

**Pull request merged**